from pybiz.exceptions import PybizError


class ApplicationError(PybizError):
    pass
